### DISCUSSION TITLE
Web Platform Installer \ Wordpress direct link returns error 404

### DISCUSSION THREAD

**Discussion by mjariri** [community/community#36372] [Actions]
Hi,
We are trying to install the WordPress on IIS using the web platform installer but it's getting stuck, and when we check the direct download link we receive the error 404 on the GitHub page. 

anybody can help with this?

Thanks.

**Comment by ossanna16** [comment]
Hey there @mjariri and thank you for asking a good question! I'm glad you found our community 🙂

**Comment by CharlesRakesh** [comment]
https://github.com/CharlesRakesh/Elementor-Pro-Form/issues/1
